test/inductor/test_distributed_patterns.py

```diff
@@ -63,13 +63,13 @@ def bw_post_hook(mod, gI, gO):
         mod.weight.grad = new_grad
 
     torch.manual_seed(1234)
-    m = nn.Linear(20, 10, bias=False)
+    m = nn.Linear(20, 10, bias=False).to(device="cuda")
     m.weight = nn.Parameter(reduce_scatter(m.weight))
     m.register_full_backward_pre_hook(bw_pre_hook)
     m.register_full_backward_hook(bw_post_hook)
     m.register_forward_pre_hook(fw_pre_hook)
     m.register_forward_hook(fw_post_hook)
-    return m, torch.rand(2, 20, requires_grad=True)
+    return m, torch.rand(2, 20, requires_grad=True, device="cuda")
```

Comment:
What's going on here?

Reply:
oh welp, accidental commit - my env was broken in a way that broken inductor cpu compilation so I was testing with cuda (these tests were also for my ppFSDP testing, not relevant to this PR stack).